src/lib/features/export-import-toggles/export-import.e2e.test.ts

```diff
@@ -249,14 +316,15 @@ test('exports features', async () => {
         ],
         segments: [
             {
+                id: segment.id,
```

Comment:
Id is also part of the export and used to link the strategy to the segment name. The lack of this value in the test doesn't fail because `toMatchObject` checks only a subset: https://jestjs.io/docs/expect#tomatchobjectobject